observability/correlation-id.md

```diff
@@ -2,7 +2,7 @@
 
 ## The Need
 
-In a distributed system architecture (microservice architecture), it is highly difficult to understand a single end to end customer transaction flow through the various components.
+In a distributed system architecture (micro service architecture), it is highly difficult to understand a single end to end customer transaction flow through the various components.
```

Comment:
I would argue that "microservice" as one word is how it is most often referred to....